articles/automation-testing-in-selenium/index.md

```diff
@@ -0,0 +1,99 @@
+Digital revolution has led to an increase in the number of web applications. Developers are constantly creating new web applications to meet various demands. Before these applications are commissioned for use, developers test them (using automation testing tools) to ensure that they function without problems. Selenium is an effective automation tool that can help developers to validate their applications. 
+
+This article will take you through fundamental aspects of this software. It provides an overview of how automation testing takes place in Selenium software. The article has also highlighted the main types of testing done by Selenium.  
+
+### Introduction to Selenium Automation Testing
+Selenium is a software used for automation testing in new applications. This software provides a framework for the testing of web applications (across various web platforms). It is open-source, which means that developers will not incur costs related to licensing. 
+
+Automation testing in Selenium has helped in solving some of the challenges experienced in manual testing. In manual testing, the test cases are executed manually. The outcome of the test is compared with the expected or desired outcome. The disparity between the desired and actual outcome indicates the presence of a defect. This defect is fixed to ensure that the application is error-free. 
+
+Selenium conducts automation testing within a shorter time compared to manual testing. It also provides various tools that increase the scope of testing. In this type of automation testing, logs are created automatically, which increases the level of accuracy. 

```

Comment:
> In this type of automation testing, logs are created automatically, which increases the level of accuracy. 

How do automatic logs increase the level of accuracy?

If you explain this briefly, it would give a better intuition to the reader.

Reply:
I have revised the sentence.